packages/epics/src/agreements/components/token-payout-field-array.stories.tsx

```
@@ -0,0 +1,36 @@
+      },
+      {
+        icon: 'https://raw.githubusercontent.com/spothq/cryptocurrency-icons/refs/heads/master/svg/icon/eth.svg',
+        symbol: 'ETH',
+        name: 'Ethereum',
+      },
+    ],
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    expect(canvas.getByText(/Payment Request/i)).toBeTruthy();
+    expect(canvas.getByText(/\Add/i)).toBeTruthy();
```

ESLint

[error] Unnecessary escape character: \A.

(no-useless-escape)